### PR TITLE
Feature/map

### PR DESCRIPTION
**Pull request by ptesser**
### What type of request did you want to push:

- [x] Feature
- Bug
- Changelog
- Other

### Which is the new behaviour:

- https://trello.com/c/XsmaF3KO
- https://trello.com/c/FUXmOTRr

### Does this request introduce a breaking-change?

- [x] yes
- no
- maybe

### Other information:

### CODE REVIEW

File: app/src/models/position.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export interface Position {
+  latitude: number;
+  longitude: number;
+}
```
Comment (Vannnu) [inline app/src/models/position.ts]: missing EOF

Comment (ptesser) [reply app/src/models/position.ts]: Fixed

File: app/src/containers/FountainsMap/FountainsMap.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    ...StyleSheet.absoluteFillObject,
+  },
+});
+
+const FountainsMap = () => {
+
+  const [error, setError] = useState('');
+  const [position, setPosition]: [Position, Dispatch<SetStateAction<Position>>] = useState({
+    latitude: 0,
+    longitude: 0
```
Comment (Vannnu) [inline app/src/containers/FountainsMap/FountainsMap.tsx]: missing comma

File: app/src/containers/FountainsMap/FountainsMap.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        }}>
+        { /** FIXME: Change icon using `image` prop with an human placeholder */ }
+        <Marker coordinate={position}></Marker>
+        { /** TODO: Iterate over all fountains */}
+      </MapView>
+    </View>
+  );
+};
+
+export default FountainsMap;
```
Comment (Vannnu) [inline app/src/containers/FountainsMap/FountainsMap.tsx]: missing EOF

**Pull request merged**